pyproject.toml

```diff
@@ -73,81 +73,81 @@ docs = ["recommonmark", "sphinx", "sphinx-click", "sphinx-rtd-theme", "sphinxcon
 cleanup = true
 entropy = true
 exclude-path-patterns = [
-    'poetry.lock',
-    'pyproject.toml',
+    {path-pattern = 'poetry.lock'},
+    {path-pattern = 'pyproject.toml'},
     # To not have to escape `\` in regexes, use single quoted
     # TOML 'literal strings'
-    '\.github/workflows/(.*)\.yml',
-    'docs/source/(.*)\.rst',
-    'tests/test_base_scanner\.py',
-    'tests/test_util.py',
-    'tests/data/testRules.json',
-    'tartufo/data/default_regexes\.json',
-    '\.git/.*',
+    {path-pattern = '\.github/workflows/(.*)\.yml'},
+    {path-pattern = 'docs/source/(.*)\.rst'},
+    {path-pattern = 'tests/test_base_scanner\.py'},
+    {path-pattern = 'tests/test_util.py'},
+    {path-pattern = 'tests/data/testRules.json'},
+    {path-pattern = 'tartufo/data/default_regexes\.json'},
+    {path-pattern = '\.git/.*'},
```

Comment:
Since we're in here touching these, can we go ahead and make sure all the `.`s are escaped?

Reply:
Done!